containers/build_backend_platforms.go

```diff
@@ -9,7 +9,6 @@ import (
 type DistroBuildOptsFunc func(executil.Distribution, *BuildInfo) *executil.GoBuildOpts
 
 var DefaultTags = []string{
-	"netgo",
```

Comment:
This sounds unrelated. Why don't we no longer use Go's network layer by default?